ajo/PersonalizationPayloads.md

```diff
@@ -0,0 +1,396 @@
+## Personalization Payloads
```

Comment:
I'd call it Sample Adobe Experience Platform Edge Network Personalization Payloads